collector.js

```diff
@@ -37,17 +37,21 @@ const COLLECTOR_RETRY_OPTS = {
  * @param {String} [alOptional.alApiEndpoint] - (optional) Alert Logic API endpoint. Default is process.env.CUSTOMCONNSTR_APP_AL_API_ENDPOINT
  * @param {String} [alOptional.alIngestEndpoint] - (optional) Alert Logic Ingestion service endpoint. Default is process.env.APP_INGEST_ENDPOINT
  * @param {String} [alOptional.alDataResidency] - (optional) data residency on Alert Logic side. Default is process.env.CUSTOMCONNSTR_APP_AL_RESIDENCY
- *
+ * @param {String} [alOptional.filterJSON] - (optional) Alert Logic filterJSON. Default is process.env.CUSTOMCONNSTR_APP_AL_FILTERJSON
+ * @param {String} [alOptional.filterREGEX] - (optional) Alert Logic filterREGEX. Default is process.env.CUSTOMCONNSTR_APP_AL_FILTERREGEX
+ * 
  */
 class AlAzureCollector {
     constructor(azureContext, collectorType, version,
-            {hostId, sourceId, aimsKeyId, aimsKeySecret, alApiEndpoint, alIngestEndpoint, alDataResidency} = {}) {
+            {hostId, sourceId, aimsKeyId, aimsKeySecret, alApiEndpoint, alIngestEndpoint, alDataResidency, filterJSON, filterREGEX} = {}) {
         this._azureContext = azureContext;
         this._collectorType = collectorType;
         this._version = version;
         
         this._hostId = hostId ? hostId : process.env.COLLECTOR_HOST_ID;
         this._sourceId = sourceId ? sourceId : process.env.COLLECTOR_SOURCE_ID;
+        this._filterJson = filterJSON ? filterJSON : process.env.CUSTOMCONNSTR_APP_AL_FILTERJSON;
+        this._filterRegex = filterREGEX ? filterREGEX : process.env.CUSTOMCONNSTR_APP_AL_FILTERREGEX;
```

Comment:
These are not connection string parameters. They should be named like `COLLECTOR_FILTER_JSON, COLLECTOR_FILTER_REGEX`

Reply:
Done